tests/conftest.py

```diff
@@ -41,6 +41,10 @@ def ensure_system(cmd):
 def pytest_addoption(parser):
     parser.addoption("--dvsname", action="store", default=None,
                       help="dvs name")
+    
+    parser.addoption("--forcedvs", action="store_true", default=False,
+                      help="force persistent dvs when ports < 32 ")
+ 
```

Comment:
```suggestion
    parser.addoption("--forcedvs", action="store_true", default=False,
                      help="force persistent dvs when ports < 32")
```
I'd remove the extra space before and after this option + in the help string to keep it consistent.